File: test/src/js/SceneApp.js
// SceneApp.js

var bongiovi = require("./libs/bongiovi");
var GL = bongiovi.GL;

function SceneApp() {
	bongiovi.Scene.call(this);
}


var p = SceneApp.prototype = new bongiovi.Scene();


p._initViews = function() {
	console.log('Init Views');
	this._vAxis = new bongiovi.ViewAxis();
};


p._initTextures = function() {
	console.log('Init Textures');
};


p.render = function() {
	var grey = .11;
	GL.clear(grey, grey, grey, 1.0);

	this._vAxis.render();
};

module.exports = SceneApp;

File: cinder/src/SceneApp.cpp
//
//  SceneApp.cpp
//  cinder
//
//  Created by Li Yi-Wen on 03/06/2015.
//
//

#include "SceneApp.h"

SceneApp::SceneApp(app::WindowRef window) : Scene(window) {
    initTextures();
    initViews();
}


void SceneApp::initTextures() {
    cout << "Init Textures" << endl;
}

void SceneApp::initViews() {
    cout << "Init Views" << endl;
    
    _vAxis = new ViewAxis();
}

void SceneApp::render() {
//    gl::clear(Colorf(.96f, .96f, .96f));
    
    _vAxis->render();
}